Tools/CISupport/ews-build/config.json

```diff
@@ -370,6 +374,13 @@
       "triggered_by": ["gtk-build-ews"],
       "workernames": ["igalia3-gtk-wk2-ews", "igalia4-gtk-wk2-ews", "igalia14-gtk-wk2-ews", "igalia15-gtk-wk2-ews"]
     },
+    {
+      "name": "API-Tests-WPE-EWS", "shortname": "api-wpe", "icon": "testOnly",
+      "factory": "APITestsFactory", "platform": "wpe",
+      "configuration": "release", "architectures": ["x86_64"],
+      "triggered_by": ["wpe-build-ews"],
+      "workernames": ["igalia12-wpe-ews", "igalia13-wpe-ews", "igalia14-wpe-ews", "igalia12-wpe-ews"]
```

Comment:
igalia12-wpe-ews is duplicated here (instead of igalia15-wpe-ews",)

Reply:
Good catch! thanks!